recipes/arraymorph/meta.yaml

```diff
@@ -0,0 +1,72 @@
+package:
+  name: arraymorph
+  version: 0.1.1
+
+source:
+  url: https://github.com/ICICLE-ai/ArrayMorph/archive/refs/tags/v0.1.1.tar.gz
+  sha256: 833651c4440971cc52469c18d8b8cfa521080a085e09661be5027ecbbae9ddca
+
+build:
+  skip: true  # [not linux64]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('arraymorph', max_pin='x.x') }}
+  script: |
+    #!/bin/bash
+    # Build from source code
+    cd "${SRC_DIR}/arraymorph/" || exit 1
+    cmake -B ./build -S . -DCMAKE_PREFIX_PATH=$BUILD_PREFIX
```

Comment:
https://conda-forge.org/docs/maintainer/knowledge_base/#using-cmake

We provide an environment variable %CMAKE_ARGS% that sets all the appropriate flags for our build environments including prefix paths, compiler directories etc.